CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
-NEXT
+1.1.2 / 2021/04/10
 ====================
-- **DEPRECATED:** `utils.waitForRunToFinish` please use `apify-client` and its `waitForFinish` functions. Sorry, forgot to deprecate this with v1 release.
+- **DEPRECATED:** `utils.waitForRunToFinish` please use the `apify-client` package and its `waitForFinish` functions. Sorry, forgot to deprecate this with v1 release.
+- Fix internal `require` that broke the SDK with `underscore` 1.13 release.
```

Comment:
see, this would never happen with pinned version, this is exactly the type of problems you can get around/have control over. with lock file it would also be better - we would get a PR update that would show us the failure right after the release (if we have such case covered in tests).

tbh its feels weird to depend on underscore nowadays, es6 is in most cases enough (literally never needed to use underscore/lodash in last 3 years myself), but that's another story (will be happy to graduallity try to remove it - less deps we have, less issues like this we will have)

Reply:
I was expecting this for a few hours now 😄

Yeah, we need to review those. The dependency on `underscore` is mostly because we historically have an `injectUnderscore()` function to inject it into a browser.